subprojects/internal-testing/src/main/groovy/org/gradle/test/fixtures/file/TestFile.java

```diff
@@ -732,5 +732,13 @@ public Snapshot(long modTime, HashCode hash) {
             this.modTime = modTime;
             this.hash = hash;
         }
+
+        public long getModTime() {
+            return modTime;
+        }
+
+        public HashCode getHash() {
+            return hash;
+        }
```

Comment:
Is this necessary?

Reply:
Well, I am accessing the `hash` from the test, so I guess it is. I don't like the warning I get when access the `private` field.

Reply:
Sorry, I missed the `hash`, I was looking for `modTime`.